guides/training_keras_models_on_cloud.py

```diff
@@ -1,421 +1,244 @@
 """
 Title: Training Keras models with TensorFlow Cloud
-Author: [Jonah Kohn](https://jonahkohn.com)
+Authors: [Jonah Kohn](https://jonahkohn.com)[Sina Chavoshi](https://www.linkedin.com/in/sinachavoshi/)
 Date created: 2020/08/11
-Last modified: 2020/08/11
-Description: In-depth usage guide for TensorFlow Cloud.
+Last modified: 2021/07/23
+Description: Usage guide for TensorFlow Cloud.
 """
 """
 ## Introduction
 
-[TensorFlow Cloud](https://github.com/tensorflow/cloud) is a Python package that
-provides APIs for a seamless transition from local debugging to distributed training
-in Google Cloud. It simplifies the process of training TensorFlow models on the
-cloud into a single, simple function call, requiring minimal setup and no changes
-to your model. TensorFlow Cloud handles cloud-specific tasks such as creating VM
-instances and distribution strategies for your models automatically. This guide
-will demonstrate how to interface with Google Cloud through TensorFlow Cloud,
-and the wide range of functionality provided within TensorFlow Cloud. We'll start
-with the simplest use-case.
-"""
+<table class="tfo-notebook-buttons" align="left">
```

Comment:
Remove the table. Links to colab / etc are autogenerated.

Reply:
Done.